clients/src/test/java/org/apache/kafka/clients/MetadataTest.java

```diff
@@ -77,6 +79,11 @@ private static MetadataResponse emptyMetadataResponse() {
                 Collections.emptyList());
     }
 
+    private <T> void assertEqualCollections(Collection<T> expected, Collection<T> actual) {
```

Comment:
nit: this name seems misleading since order could be important for arbitrary collections. Since we only have a couple uses, maybe we can get rid of it and use `assertEquals(new HashSet(A), new HashSet(B))` for example.